sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -203,6 +203,17 @@ export class StreamingReceiver extends MessageReceiver {
           sbError,
           `${this.logPrefix} 'receiver_error' event occurred. The associated error is`
         );
+        if (
+          sbError?.code &&
+          ["MessagingEntityDisabled", "MessagingEntityNotFound"].includes(sbError.code)
```

Comment:
super nitpicky: perhaps do 2 checks instead of 3?
```suggestion
         sbError.code === "MessagingEntityDisabled" || sbError.code === "MessagingEntityNotFound"
```

Reply:
Yeah, I was trying to leave room for potential future errors.